script.js

```diff
@@ -658,4 +658,13 @@
     }
   });
 
+  // Custom
+
+  window.addEventListener("DOMContentLoaded", () => {
+    //Hide Contact Us element on new request page
+    if (location.pathname.indexOf("/requests/new") > -1) {
+      document.getElementById("contact-us-element").style.display = "none";
+    }
+  });
+
```

Comment:
I believe we can remove this duplicate code as it is added in `custom.js`.

Reply:
No, we can not remove this from `script.js` as the Zendesk theme now starts using [Rollup to compile the JS and CSS](https://github.com/mitodl/bootcamps-zendesk-theme?tab=readme-ov-file#stylesheet-and-javascript) files that are used in the theme, into `style.css` and `script.js`. They intentionally did not use babel when compiling script.js so we can get a clean bundle output. We have to make sure now on-ward - Do not edit `style.css` and `script.js` directly.